app/services/ccms/attribute_value_generator.rb

```diff
@@ -105,6 +105,14 @@ def used_delegated_functions_on(_options)
       legal_aid_application.used_delegated_functions_on.strftime("%d-%m-%Y")
     end
 
+    def backdated_sca_application?(_options)
+      legal_aid_application.special_children_act_proceedings? && legal_aid_application.used_delegated_functions?
+    end
+
+    def delegated_function_or_backdated_sca?(_options)
```

Comment:
Is this not the same as `legal_aid_application.used_delegated_functions?`
Are you adding this method because that's bad terminology for SCA?

Reply:
Yes, this was from an earlier version of code that was superfluous after refactoring